src/components/Dropdown/Dropdown.jsx

```diff
@@ -249,6 +250,7 @@ const Dropdown = ({
       tabIndex={tabIndex}
       id={id}
       autoFocus={autoFocus}
+      closeMenuOnSelect={!keepDropdownOpenOnSelection}
```

Comment:
currently it's being closed automatically, so to prevent breaking, we pass with "true" and enable it